Resources/Prototypes/_NF/Objects/Devices/admin_hand_teleporter.yml

```diff
@@ -0,0 +1,16 @@
+- type: entity
+  id: AdminHandTeleporter
+  parent: BaseItem
+  name: hand teleporter [admeme]
```

Comment:
Use suffix

Also please use main existing items as parents you dont need this 200 lines just for

```
  - type: Portal
    canTeleportToOtherMaps: true
```